src/devices/bus/technics/hdae5000.cpp

```diff
@@ -0,0 +1,110 @@
+// license:BSD-3-Clause
+// copyright-holders:Antoine Mine, Olivier Galibert, Felipe Sanches
+//
+// HD-AE5000, Hard Disk & Audio Extension for Technics KN5000 emulation
+//
+// The HD-AE5000 was an extension board for the Technics KN5000 musical keyboard.
+// It provided a hard-disk, additional audio outputs and a serial port to interface
+// with a computer to transfer files to/from the hard-drive.
+
+#include "emu.h"
+#include "bus/ata/hdd.h"
+#include "hdae5000.h"
+#include "machine/i8255.h"
```

Comment:
The modules own header should come immediately after the prefix header, before headers for other things it uses.

Reply:
What do you mean by "the prefix header"? Are you referring to the "emu.h" one?

Reply:
https://docs.mamedev.org/contributing/cxx.html#structural-organization